doc/source/configuration/monitoring.rst

```diff
@@ -0,0 +1,22 @@
+===========
+Monitoring
+===========
+
+SMART Drive Monitoring
+=======================
+After pulling in the latest changes into your local kayobe config, reconfigure prometheus and Grafana
```

Comment:
```suggestion
After pulling in the latest changes into your local kayobe config, reconfigure Prometheus and Grafana
```

Reply:
... to deploy the SMART drive monitoring Grafana dashboard and Prometheus rules.